pkg/epp/metrics/metrics.go

```diff
@@ -35,42 +35,39 @@ const (
 
 var (
 	// Inference Model Metrics
-	requestCounter = compbasemetrics.NewCounterVec(
-		&compbasemetrics.CounterOpts{
-			Subsystem:      InferenceModelComponent,
-			Name:           "request_total",
-			Help:           "Counter of inference model requests broken out for each model and target model.",
-			StabilityLevel: compbasemetrics.ALPHA,
```

Comment:
Can you keep the metric stage for all metrics?

Reply:
stability level is just added to the help message of the metric: https://github.com/kubernetes/component-base/blob/88ca0abd83492f6f11c502946dd2657f8b3374b4/metrics/opts.go#L149-L153

To align with other k8s metrics https://kubernetes.io/docs/reference/instrumentation/metrics/ can you add this back manually to add back the stability level?

Reply:
Per discussed in other comments, can you please just add the stability level to the help message of the metric? Thanks!

Reply:
It's is too complicated and unnecessary to change `&compbasemetrics.CounterOpts` to `prometheus.CounterOpts`. I'll revert this change and only retain the logic for merging two metric servers. I can open another PR to change this struct If you think it's necessary.